### ISSUE TITLE
Push generated XML to GitHub from the web app

### ISSUE THREAD

**Issue by code-geek** [NASA-IMPACT/sde-indexing-helper#195]
```[tasklist]
### Acceptance Criteria
- [x] For any given collection, pull the latest config from the `dev` branch
- [x] Add rules to the config file from the web app (depends on #161)
- [x] Generate a name for the branch
- [x] Check if the branch already exists, and create it if not
- [x] Create a new pull request to the `dev` branch
- [x] Provide a link to the pull request to the user
- [x] Provide a mechanism to bulk push rules for multiple collections to GitHub
- [ ] https://github.com/NASA-IMPACT/sde-indexing-helper/issues/227
```
